routes/api_nyt.js

```diff
@@ -0,0 +1,39 @@
+// eslint-disable-next-line new-cap
+const router = require('express').Router();
+const { nytCounties, nytStates, nytNationwide } = require('../utils/nyt_cache');
+
+router.get('/v2/nyt/states', async (req, res) => {
+	const data = nytStates();
+	const { state: queryState } = req.query;
```

Comment:
to fit the other endpoints, move object deconstruction for query parameters to the first line, then initialize data